style.css

```diff
@@ -205,7 +205,7 @@ body.lock {
 }
 .form-search input {
     all: unset;
-    font-size: 12px;
+    font-size: 4vw;
```

Comment:
I highly doubt that 4vw is 12px.

Reply:
a little more, I decided it looked better

Reply:
Changed all sizes to 3.5 vw to be consistent with each other

Reply:
Went back to px